conceptnet5/vectors/transforms.py

```diff
@@ -101,3 +148,62 @@ def make_replacements(small_frame, big_frame, limit=100000):
             if got:
                 replacements[term] = got[0]
     return replacements
+
+
+def choose_small_vocabulary(big_frame):
+    """
+    Choose the vocabulary of the small frame, by eliminating the terms which:
+     - contain more than one word
+     - are not in ConceptNet
+    """
+    DATA = environ.get("CONCEPTNET_BUILD_DATA", "data")
+    concepts = set(line.strip() for line in open(path.join(DATA, 'stats', 'core_concepts.txt')))
```

Comment:
This filename should be passed in as a parameter -- that way Snakemake can keep track of it and make sure that the file exists before we do this step.